src/app/rpmostree-builtin-status.c

```diff
@@ -283,21 +287,24 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
 
   const char *policy = rpmostree_sysroot_get_automatic_update_policy (sysroot_proxy);
 
-  g_print ("State: %s", txn_proxy ? "busy" : "idle");
+  g_print ("State: %s\n", txn_proxy ? "busy" : "idle");
+  g_print ("Automatic Updates: ");
```

Comment:
Minor/bikeshed: lowercase `updates` to make consistent with "Available update"? Alternatively, convert both this one and "Available update" to systemd/golang naming like `AutomaticUpdates` and `AvailableUpdate`?